tests/vmi_lifecycle_test.go

```diff
@@ -569,6 +569,63 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				tests.WaitForSuccessfulVMIStart(newVMI)
 			})
+
+			It("VMIs with Bridge Networking shouldn't fail after the kubelet restarts", func() {
+				bridgeVMI := vmi
+				// Remove the masquerade interface to use the default bridge one
+				bridgeVMI.Spec.Domain.Devices.Interfaces = nil
+				bridgeVMI.Spec.Networks = nil
+				v1.SetDefaults_NetworkInterface(bridgeVMI)
+				Expect(bridgeVMI.Spec.Domain.Devices.Interfaces).NotTo(BeEmpty())
+
+				By("starting a VMI with bridged network on a node")
+				bridgeVMI, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(bridgeVMI)
+				Expect(err).To(BeNil(), "Should submit VMI successfully")
+
+				// Start a VirtualMachineInstance with bridged networking
+				nodeName := tests.WaitForSuccessfulVMIStart(bridgeVMI)
+
+				tests.VerifyDummyNicForBridgeNetwork(bridgeVMI)
+
+				By("restarting kubelet")
+				pod := renderPkillAllPod("kubelet")
+				pod.Spec.NodeName = nodeName
+				_, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(pod)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("starting another VMI on the same node, to verify kubelet is running again")
+				newVMI := newCirrosVMI()
+				newVMI.Spec.NodeSelector = map[string]string{"kubernetes.io/hostname": nodeName}
+				Eventually(func() error {
+					newVMI, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(newVMI)
+					Expect(err).To(BeNil())
+					return nil
+				}, 100, 10).Should(Succeed(), "Should be able to start a new VM")
+
+				By("checking if the VMI with bridged networking is still running, it will verify the CNI didn't cause the pod to be killed")
+				tests.WaitForSuccessfulVMIStart(bridgeVMI)
+			})
+
+			It("VMIs with Bridge Networking should work with Duplicate Address Detection (DAD)", func() {
+				bridgeVMI := vmi
+				// Remove the masquerade interface to use the default bridge one
+				bridgeVMI.Spec.Domain.Devices.Interfaces = nil
+				bridgeVMI.Spec.Networks = nil
+				v1.SetDefaults_NetworkInterface(bridgeVMI)
+				Expect(bridgeVMI.Spec.Domain.Devices.Interfaces).NotTo(BeEmpty())
+
+				By("starting a VMI with bridged network on a node")
+				bridgeVMI, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(bridgeVMI)
+				Expect(err).To(BeNil(), "Should submit VMI successfully")
+
+				// Start a VirtualMachineInstance with bridged networking
+				By("Waiting the VirtualMachineInstance start")
+				tests.WaitUntilVMIReady(bridgeVMI, console.LoginToAlpine)
+				tests.VerifyDummyNicForBridgeNetwork(bridgeVMI)
+				console.RunCommand(bridgeVMI, "/usr/sbin/ip a", 200*time.Second)
+				Expect(console.RunCommand(bridgeVMI, "/usr/sbin/arping -D -I k6t-eth0 -c 2 $(hostname -i) | grep Received | cut -d ' ' -f 2", 200*time.Second)).To(Equal(0))
```

Comment:
1. I think it will be safer to get the vm ip using-
`vmIp = libnet.GetVmiPrimaryIpByFamily(bridgeVMI, ipFamily)`
On my laptop `hostname -i` returns 127.0.0.1. On a dual stack vm it may return more than one address. I would prefer not to depend on the /etc/host and take a safer approach.
2. Why did you change etho to k6t-eth0? The vm is not familiar with k6t-eth0?

Reply:
I'm just trying something since `eth0` was down inside the vm

Reply:
> vmIp = libnet.GetVmiPrimaryIpByFamily(bridgeVMI, ipFamily)
>
that's what I was looking for, thanks.

Reply:
`arping: interface eth0 is down`.   I'll keep poking at it.

Reply:
Try to use cirros instead of alpine.